### ISSUE TITLE
bitbucket pipelines not working using jovo v2

### ISSUE THREAD

**Issue by rjolayolay** [jovotech/jovo-framework#331]
- [ x] Bug report

## Expected Behavior
what should happen = i update code base to jovo v2, i deploy my app, my pipeline to bitbucket works seamlessly and all tests on bitbucket pass

## Current Behavior
sooooo our app is using gulp 4.0, that won't work w/ <jovo deploy> to lamba in bitcucket pipeline, we attempted to downgrade to 3.9, it worked locally, it worked on bitbucket pipelines, but it won't deploy, tests don't pass in bitbucket pipeline.

### Error log

```
[0] TypeError: _.get is not a function
[0]     at Project.getStage (/opt/atlassian/pipelines/agent/build/node_modules/jovo-cli-core/dist/src/Project.js:360:19)
[0]     at CommandInstance.vorpalCommand.action (/opt/atlassian/pipelines/agent/build/node_modules/jovo-cli/dist/commands/run.js:49:31)
[0]     at <anonymous>
[0]  [1D[1CUnhandled Rejection at: Promise Promise {
[0]   <rejected> Error: true
[0]     at Vorpal.<anonymous> (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:168:17)
[0]     at callback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:830:22)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:961:7
[0]     at Session._commandSetCallback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:440:5)
[0]     at Session.session.completeCommand (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:520:12)
[0]     at onCompletion (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:450:10)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:481:7
[0]     at <anonymous> } reason: Error: true
[0]     at Vorpal.<anonymous> (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:168:17)
[0]     at callback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:830:22)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:961:7
[0]     at Session._commandSetCallback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:440:5)
[0]     at Session.session.completeCommand (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:520:12)
[0]     at onCompletion (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:450:10)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:481:7
[0]     at <anonymous>
[0] Unhandled Rejection at: Promise Promise {
[0]   <rejected> Error: true
[0]     at Vorpal.<anonymous> (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:168:17)
[0]     at callback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:830:22)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:961:7
[0]     at Session._commandSetCallback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:440:5)
[0]     at Session.session.completeCommand (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:520:12)
[0]     at onCompletion (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:450:10)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:481:7
[0]     at <anonymous> } reason: Error: true
[0]     at Vorpal.<anonymous> (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:168:17)
[0]     at callback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:830:22)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:961:7
[0]     at Session._commandSetCallback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:440:5)
[0]     at Session.session.completeCommand (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:520:12)
[0]     at onCompletion (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:450:10)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:481:7
[0]     at <anonymous>
```
## Environment
* Operating System: Linux

this is the package.json

```{ "name": "jovo-sample-voice-app-nodejs",
 "version": "2.0.2",
 "description": "A sample voice app that works with the Jovo Framework",
 "main": "src/index.js",
 "dependencies": {
   "jovo-cms-googlesheets": "^2.0.4",
   "jovo-db-filedb": "^2.0.2",
   "jovo-framework": "^2.0.4",
   "jovo-platform-alexa": "^2.0.2",
   "jovo-platform-googleassistant": "^2.0.2",
   "jovo-plugin-debugger": "^2.0.2"
 },
 "devDependencies": {
   "concurrently": "^4.1.0",
   "firebase-admin": "^6.4.0",
   "gulp": "^3.9.1",     //~~~~  <-----altered version ~~~~
   "gulp-install": "^1.1.0",
   "gulp-run-command": "0.0.9",   
   "gulp-zip": "^4.2.0",
   "jest": "^23.6.0",
   "twilio": "^3.26.1",
   "wait-on": "^3.2.0"
 },
 "scripts": {
   "tsc": "node -v",
   "run-and-test": "concurrently \"jovo run\" \"npm test\" -k -s first",
   "test": "wait-on tcp:3000 && jest",
   "bundle": "gulp --gulpfile node_modules/jovo-framework/gulpfile.js --cwd ./",
   "start": "cd src && node index.js --webhook",
   "launch": "npm start -- --launch"
 },
 "repository": {
   "type": "git",
   "url": "git+https://github.com/jovotech/jovo-sample-voice-app-nodejs.git"
 },
 "author": "Jovo",
 "license": "Apache-2.0",
 "bugs": {
   "url": "https://github.com/jovotech/jovo-sample-voice-app-nodejs/issues"
 },
"homepage": "https://github.com/jovotech/jovo-sample-voice-app-nodejs#readme"
}```




```
## Full Pipeline Log

```
+ umask 000

+ GIT_LFS_SKIP_SMUDGE=1 git clone --branch="feature/jovo2" --depth 50 https://x-token-auth:$REPOSITORY_OAUTH_ACCESS_TOKEN@bitbucket.org/cleanablemydrycleaner/my-dry-cleaner-v2.git $BUILD_DIR
Cloning into '/opt/atlassian/pipelines/agent/build'...

+ git reset --hard 13c22fff50cf4e9dde76354d5fdc9fe15b480597
HEAD is now at 13c22fff started from clean template

+ git config user.name bitbucket-pipelines

+ git config user.email commits-noreply@bitbucket.org

+ git config push.default current

+ git config http.${BITBUCKET_GIT_HTTP_ORIGIN}.proxy http://localhost:29418/

+ git remote set-url origin http://bitbucket.org/cleanablemydrycleaner/my-dry-cleaner-v2

+ chmod 777 $BUILD_DIR

Cache "node": Downloading
Cache "node": Downloaded 22.9 MiB in 3 seconds
Cache "node": Extracting
Cache "node": Extracted in 1 seconds
+ npm install
npm WARN notice [SECURITY] minimatch has the following vulnerability: 1 high. Go here for more details: https://nodesecurity.io/advisories?search=minimatch&version=0.2.14 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] minimatch has the following vulnerability: 1 high. Go here for more details: https://nodesecurity.io/advisories?search=minimatch&version=2.0.10 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.
npm WARN notice [SECURITY] lodash has the following vulnerability: 1 low. Go here for more details: https://nodesecurity.io/advisories?search=lodash&version=1.0.2 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.

> grpc@1.17.0 install /opt/atlassian/pipelines/agent/build/node_modules/grpc
> node-pre-gyp install --fallback-to-build --library=static_library

node-pre-gyp WARN Using request for node-pre-gyp https download 
[grpc] Success: "/opt/atlassian/pipelines/agent/build/node_modules/grpc/src/node/extension_binary/node-v57-linux-x64-glibc/grpc_node.node" is installed via remote
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 762 packages, removed 47 packages and updated 121 packages in 15.861s

+ npm install jovo-cli
npm WARN notice [SECURITY] lodash has the following vulnerability: 1 low. Go here for more details: https://nodesecurity.io/advisories?search=lodash&version=3.10.1 - Run `npm i npm@latest -g` to upgrade your npm version, and then `npm audit` to get more info.

> nodemon@1.18.9 postinstall /opt/atlassian/pipelines/agent/build/node_modules/nodemon
> node bin/postinstall || exit 0

[32mLove nodemon? You can now support the project via the open collective:[22m[39m
 > [96m[1mhttps://opencollective.com/nodemon/donate[0m

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ jovo-cli@2.0.6
added 365 packages and removed 2 packages in 19.298s

+ npm install jovo-framework
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ jovo-framework@2.0.5
added 70 packages, removed 2 packages and updated 2 packages in 10.506s

+ npm install firebase-admin
+ firebase-admin@6.4.0
added 67 packages, removed 2 packages and updated 1 package in 13.781s
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})


+ npm install --save jovo-cms-googlesheets
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ jovo-cms-googlesheets@2.0.4
added 67 packages, removed 2 packages and updated 1 package in 14.174s

+ npm install jest
npm WARN deprecated kleur@2.0.2: Please upgrade to kleur@3 or migrate to 'ansi-colors' if you prefer the old syntax. Visit <https://github.com/lukeed/kleur/releases/tag/v3.0.0\> for migration path(s).
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ jest@23.6.0
added 67 packages, removed 2 packages and updated 1 package in 16.454s

+ npm install wait-on
npm WARN deprecated hoek@5.0.4: This version is no longer maintained. Please upgrade to the latest version.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ wait-on@3.2.0
added 67 packages, removed 2 packages and updated 1 package in 11.85s

+ npm install concurrently
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

+ concurrently@4.1.0
added 67 packages, removed 2 packages and updated 1 package in 12.248s

+ npm run-script run-and-test

> jovo-sample-voice-app-nodejs@2.0.2 run-and-test /opt/atlassian/pipelines/agent/build
> concurrently "jovo run" "npm test" -k -s first

[1] 
[1] > jovo-sample-voice-app-nodejs@2.0.2 test /opt/atlassian/pipelines/agent/build
[1] > wait-on tcp:3000 && jest
[1] 
[0]  [1D[1C[1000D[K [1D[1C
[0] TypeError: _.get is not a function
[0]     at Project.getStage (/opt/atlassian/pipelines/agent/build/node_modules/jovo-cli-core/dist/src/Project.js:360:19)
[0]     at CommandInstance.vorpalCommand.action (/opt/atlassian/pipelines/agent/build/node_modules/jovo-cli/dist/commands/run.js:49:31)
[0]     at <anonymous>
[0]  [1D[1CUnhandled Rejection at: Promise Promise {
[0]   <rejected> Error: true
[0]     at Vorpal.<anonymous> (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:168:17)
[0]     at callback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:830:22)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:961:7
[0]     at Session._commandSetCallback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:440:5)
[0]     at Session.session.completeCommand (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:520:12)
[0]     at onCompletion (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:450:10)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:481:7
[0]     at <anonymous> } reason: Error: true
[0]     at Vorpal.<anonymous> (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:168:17)
[0]     at callback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:830:22)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:961:7
[0]     at Session._commandSetCallback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:440:5)
[0]     at Session.session.completeCommand (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:520:12)
[0]     at onCompletion (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:450:10)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:481:7
[0]     at <anonymous>
[0] Unhandled Rejection at: Promise Promise {
[0]   <rejected> Error: true
[0]     at Vorpal.<anonymous> (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:168:17)
[0]     at callback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:830:22)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:961:7
[0]     at Session._commandSetCallback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:440:5)
[0]     at Session.session.completeCommand (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:520:12)
[0]     at onCompletion (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:450:10)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:481:7
[0]     at <anonymous> } reason: Error: true
[0]     at Vorpal.<anonymous> (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:168:17)
[0]     at callback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:830:22)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/vorpal.js:961:7
[0]     at Session._commandSetCallback (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:440:5)
[0]     at Session.session.completeCommand (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:520:12)
[0]     at onCompletion (/opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:450:10)
[0]     at /opt/atlassian/pipelines/agent/build/node_modules/vorpal/dist/session.js:481:7
[0]     at <anonymous>
```
```

**Comment by QuentinDela** [comment]
It looks similar to an issue I am having
https://github.com/jovotech/jovo-cli/issues/78

**Comment by jankoenig** [comment]
Hey there. Did this get resolved? We switched to `gulp 4` in a recent update + fixed jovotech/jovo-cli#78. Please reopen if the issue persists. Thanks!
